plugins/by-name/gdscript-extended-lsp/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "gdscript-extended-lsp";
+  package = "gdscript-extended-lsp-nvim";
+
+  maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  settingsExample = {
+    picker = "snacks";
```

Comment:
Don't we want to add a warning if `picker` is set to `snacks` and `plugins.snacks` is not enabled?

Reply:
The plugin itself notifies the user, if the user executes the `require("gdscript-extended-lsp").pick()` and do not have the specificied plugin

Reply:
Done